mirage/psf/psf_selection.py

```diff
@@ -291,7 +291,7 @@ def get_library_file(instrument, detector, filt, pupil, wfe, wfe_group,
             file_filt = header['FILTER'].upper()
 
             try:
-                file_pupil = header['PUPIL_MASK'].upper()
```

Comment:
Note, I don't understand how this has been working before, since PUPIL_MASK has never been a keyword that WebbPSF writes; in fact it's not even a valid keyword at all (too many characters). I think it has always been failing back to CLEAR via the exception case, whenever this function is invoked currently.

Reply:
Yeah. I must have copied the parameter name from webbpsf without thinking about it.